doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2994,62 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
+    
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
     
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
     
+Note: Since the primary purpose of Dataverse is to serve as an archive and to track data accesses and the modifications of data and metadata, a simple mechanism to delete users that have performed certain actions in the system is not provided by design. 
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation
+- Remove the user's access from all datasets and files
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
+
+Disabling a user with this endpoint will keep:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author.
+- The user's access to datafiles in the Dataverse installation, including guestbook records and acceptance of terms of use.
+- The user's account information (specifically name, email, affiliation, and position)
+
+If it is determined that disabling a user is not sufficient, it is possible anonymize a user through use of the :ref:`merge-accounts-label` feature. See :ref:`Anonymize a User <disable-a-user>` for more information about this process. 
+
+.. _anonymize-a-user:
+
+Anonymize a User
+~~~~~~~~~~~~~~~~
+
+In some cases, due to privacy laws or other organizational policies, it may be required to anonymize a user. This process should only be used in cases where a Dataverse installation has the need to completely remove the traces of a user from the Dataverse installation. This process is only provided because it is not feasible for the Dataverse Software to provide deletion and deactivation options that will satisfy all use cases and all privacy laws, so the most complete deletion option is provided here. 
+
+Note: Before going through these steps, please note that you will be permanently losing:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author. This will all be marked as "Anonymous User" in your Dataverse installation, and it will not be possible to track who made changes as part of the archival record.   
+- The user's account information (specifically name, email, affiliation, and position). This will make it impossible to follow up with the user being deleted. You may find that you need to contact a user about a security breach or something else in the future, and you will not able to do so. 
+
+You will not lose:
+
+- Email, First Name, and Last Name in Guestbook records.
```

Comment:
Ah, I see this info is here. The line above is misleading. They can be contacted.